### DISCUSSION TITLE
Execute command for argocd-server

### DISCUSSION THREAD

**Discussion by manish222agr** [argoproj/argo-cd#17301] [Q&A]
Hi,

I was looking into https://argo-cd.readthedocs.io/en/stable/operator-manual/server-commands/argocd-server/ to execute the argocd-server command.
I have argocd cli installed in my laptop.

However, I get the error as below. 

I could not find much information in the link. How can I execute this command from my laptop command window.

> argocd-server version

argocd-server: The term 'argocd-server' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

Thanks.

**Comment by FernandoMiguel** [comment]
sounds like you dont have the binary in the PATH
either fix your PATH, or export the current one, or run the binary locally with ./argocd
